doc/developers-guide/accumulators.md

```diff
@@ -61,6 +71,10 @@ Acc2 = mongoose_acc:add_prop(myprop, 123, Acc1),
 V = mongoose_acc:get_prop(myprop, Acc2).
 ```
 
+The rationale behind stripping an accumulator is that many values store in it are context-dependend.
+For example, `user` and `server` refer to the owner of the c2s process, in another one user will be different (and sometimes server).
```

Comment:
For example, in a c2s process `user` and `server` refer to its owner, while elsewhere the user will be different (or sometimes even a server). ?